superset-frontend/src/explore/components/controls/SliderControl.tsx

```diff
@@ -17,27 +17,21 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import Slider from 'src/components/Slider';
 import ControlHeader from 'src/explore/components/ControlHeader';
 
-const propTypes = {
-  onChange: PropTypes.func,
-  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+type SliderControlProps = {
+  onChange: (value: number) => void;
+  value: number;
+  default?: number;
 };
 
-const defaultProps = {
-  onChange: () => {},
-};
-
-export default function SliderControl(props) {
+export default function SliderControl(props: SliderControlProps) {
+  const { default: defaultValue, ...rest } = props;
```

Comment:
Can you add the default value for `onChange` please? I'm not sure if it's actually needed, but it won't hurt to keep it

Reply:
I think this is not required, but sure I can added it.